### PR TITLE
Add alias for np.__version__ and np.dtype

### PR DESCRIPTION
**Pull request by haojin2**
## Description ##
As title.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: python/mxnet/numpy/fallback.py [hunk 1]
```diff
@@ -106,6 +110,8 @@
     'vander',
 ]
 
+__version__ = onp.__version__
```
Comment (leezu) [inline python/mxnet/numpy/fallback.py]: Is there any mechanism to ensure `mx.np` is compatible with the `onp.__version__`? If not, should this return the version `mx.np` is compatible with?

Comment (haojin2) [reply python/mxnet/numpy/fallback.py]: At least for now the CI have 1.18 installed and we pass all current tests against that version. So, I think we're in good shape here.

Comment (leezu) [reply python/mxnet/numpy/fallback.py]: That's great. My question is, once Numpy 1.19 or later versions are released and potentially change API, should `mx.np.__version__` report `1.18`, which is the version you are currently targeting, or should it claim `1.19` which could be the numpy version installed on the users machine.

Comment (haojin2) [reply python/mxnet/numpy/fallback.py]: Once a 1.19 comes out our CI would directly report problems if any and we can fix it, I cannot predict what they're changing in the future, and the dilemma here is that the fallback-ed operators will follow the new 1.19.

Comment (leezu) [reply python/mxnet/numpy/fallback.py]: Thanks for clarifying the issue. How should this be handled for stable releases? Users may install the release at any time and the numpy version could be incompatible? Should the supported numpy version be explicitly called out and returned by `mx.np.__version__`?

Comment (haojin2) [reply python/mxnet/numpy/fallback.py]: I think it should be addressed by setting the correct dependencies in our release version. If we claim that we are compatible with 1.xx version but the pypi packages says that anything up to 1.yy is okay, then I think it's something that whoever is in charge of CD should address.

**Pull request merged**